search.go

```diff
@@ -326,7 +327,7 @@ func (l *Conn) SearchWithPaging(searchRequest *SearchRequest, pagingSize uint32)
 	searchResult := new(SearchResult)
 	for {
 		result, err := l.Search(searchRequest)
-		l.Debug.Printf("Looking for Paging Control...")
+		l.Debug.Log("Looking for Paging Control...")
```

Comment:
The log entries for this file don't provide much context.  May be out of scope for this PR since the original content is preserved, but I suspect these log entries could be made more helpful.

Reply:
Yeah agreed

Reply:
Good point. I removed the log messages from `search.go`